saltyrtc/message.py

```diff
@@ -1,30 +1,33 @@
 import abc
-import struct
-import io
 import binascii
+import io
+import struct
 
 # noinspection PyPackageRequirements
 import umsgpack
 
-from .exception import *
+from .common import sign_keys as sign_keys_
```

Comment:
It is based on [PEP8 - Function and method arguments](http://legacy.python.org/dev/peps/pep-0008/#function-and-method-arguments). I sometimes do it to avoid shadowing. There's an argument somewhere called `sign_keys` and I don't want to rename it.